apimodels/agent_models.go

```diff
@@ -129,6 +129,7 @@ type CreateHost struct {
 	// docker-related settings
 	Image                    string            `mapstructure:"image" json:"image" yaml:"image" plugin:"expand"`
 	Command                  string            `mapstructure:"command" json:"command" yaml:"command" plugin:"expand"`
+	PublishPorts             bool              `mapstructure:"publish_ports" json:"publish_ports" yaml:"publish_ports" plugin:"publish_ports"`
```

Comment:
`plugin:"expand"`

Reply:
Do we support expansions for booleans?

Reply:
No this was dumb :) I'd already taken it out though